dashboard/test/factories/factories.rb

```diff
@@ -1063,6 +1063,18 @@
     level_source {create :level_source, level: level}
   end
 
+  factory :skill do
+    key {"variables_name"}
```

Comment:
this looks like it will fail if you try to create more than one skill using this factory, due to the unique constraint on this field. take a peek at other factories to see how to make a field that auto-increments and use the same approach here.